src/gotcha_auxv.c

```diff
@@ -31,14 +31,14 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 
 static ElfW(Ehdr) *vdso_ehdr = NULL;
 static unsigned int auxv_pagesz = 0;
-
+#define BUFFER_LEN 4096
 
 int parse_auxv_contents()
 {
    char name[] = "/proc/self/auxv";
    int fd, done = 0;
-   char buffer[4096];
-   ssize_t buffer_size = 4096, offset = 0, result;
+   char buffer[BUFFER_LEN];
+   ssize_t buffer_size = BUFFER_LEN, offset = 0, result;
```

Comment:
`buffer_size` is basically `const`. so the variable should either be declared as this or replaced by `BUFFER_LEN`

Reply:
Yes thats why we added ```#define BUFFER_LEN 4096``` at the top.

Reply:
I changed the buffer_size to be a const.